site/content/kapp-controller/docs/develop/kctrl-package-authoring-of-upstream.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Packaging upstream artifacts with CLI
+---
+This tutorial explores how `kctrl` allows us to create Carvel packages using existing artifacts like manifests released as a part of a GitHub release or a Helm chart.
+For this tutorial we will package a release of `cert-manager` as a Carvel package.
+
+## Getting started
+
+To start off, let's create a directory which acts like our working directory.
+
+```bash
+$ mkdir certman-package
+$ cd certman-package
+```
+
+Next we run the `init` command to set the stage!
+
+`kctrl` asks a few quick questions to gather what it needs to know.
+We know that `cert-manager` lives on the GitHub repository _cert-manager/cert-manager_ and that it's releases have a manifest `cert-manager.yaml` which let's users deploy cert-manager on cluster. Our goal would be to build a package around this artifact.
```

Comment:
nit: I think we can move it up before `Getting started`.

Reply:
I think it helps better if closer to where the user actually inputs the values

Reply:
Would be clearer when we show the process step by step